.template/addons/bootstrap/package.json.rb

```diff
@@ -1,8 +1,4 @@
 # frozen_string_literal: true
 
-insert_into_file 'package.json', after: /"i18n-js":.+\n/ do
-  <<~JSON
-    "bootstrap": "4.5.2",
-    "bootstrap.native": "3.0.13",
-  JSON
-end
+run 'yarn add bootstrap@^4.5.2'
+run 'yarn add bootstrap.native@^3.0.13'
```

Comment:
by using Boostrap 5 which doesn't use jQuery, bootstrap native is no longer needed.